src/gf-platformid

```diff
@@ -70,6 +70,15 @@ sed -i -e 's, ignition.platform.id=[a-zA-Z0-9]*,,g' "${tmpd}"/bls.conf
 sed -i -e 's,^\(options .*\),\1 coreos.oem.id='"${platformid}"' ignition.platform.id='"${platformid}"',' "${tmpd}"/bls.conf
 coreos_gf upload "${tmpd}"/bls.conf "${blscfg_path}"
 
+if [ "$arch" = "s390x" ] && [ "$platformid" = "openstack" ]; then
```

Comment:
There's no reason to conditionalize on `platformid` here, is there?

Reply:
for openstack images we derive from qemu, so that is the only problematic case. the others don't have this problem. Hence the conditionalization.

Reply:
A lot of other images derive from qemu on x86_64, such as `aws` etc.  While I know that probably today, there's only `openstack` deriving from qemu, one could be added in the future.  It would be surprising for someone to re-discover this bug then.

In other words, this is a generic bugfix for running `zipl` after changing the bootloader arguments - it has nothing to do specifically with `openstack`, so let's drop that conditional.

Reply:
ok..makes sense.. @tuan-hoang1 agree ?

Reply:
I was thinking making a 4th parameter `runzipl=$4` to `gf-platformid` and  then

``if [ "$arch" = "s390x" ] && [ -n "$runzipl" ]; then```

But that's not necessary. Because we always have to re-run zipl anyway.